tests/edalize_common.py

```diff
@@ -1,21 +1,90 @@
-import os.path
-import tempfile
 from collections import OrderedDict
+import filecmp
+import os.path
+import shutil
+
+import pytest
 
 from edalize import get_edatool
 
+
 tests_dir = os.path.dirname(__file__)
 
+
+class TestFixture:
+    def __init__(self, test_name, tool_name, param_types, files,
+                 tool_options, ref_dir, work_root, use_vpi):
+        self.test_name = test_name
+        self.ref_dir = ref_dir
+        self.work_root = work_root
+        self.backend = _setup_backend(test_name, tool_name, param_types, files,
+                                      tool_options, work_root, use_vpi)
+
+    def compare_files(self, files, ref_subdir='.'):
```

Comment:
Please document that this function "forwards" to edalize_common.compare_files(); the docstring in there with the GOLDEN_RUN environment variable is rather useful information.